webapp/components/Badges.spec.js

```diff
@@ -2,20 +2,29 @@ import { shallowMount } from '@vue/test-utils'
 import Badges from './Badges.vue'
 
 describe('Badges.vue', () => {
-  let wrapper
+  let propsData
 
   beforeEach(() => {
-    wrapper = shallowMount(Badges, {})
+    propsData = {}
   })
 
-  it('renders', () => {
-    expect(wrapper.is('div')).toBe(true)
-  })
+  describe('shallowMount', () => {
+    const Wrapper = () => {
+      return shallowMount(Badges, { propsData })
+    }
 
-  it('has class "hc-badges"', () => {
-    expect(wrapper.contains('.hc-badges')).toBe(true)
-  })
+    it('has class "hc-badges"', () => {
+      expect(Wrapper().contains('.hc-badges')).toBe(true)
+    })
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-26T11:52:08Z" title="Wednesday, February 26th 2020, 12:52:08 pm +01:00">Feb 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,20 +2,33 @@ import { shallowMount } from '@vue/test-utils'
 import Badges from './Badges.vue'
 
 describe('Badges.vue', () => {
-  let wrapper
+  let propsData
 
   beforeEach(() => {
-    wrapper = shallowMount(Badges, {})
+    propsData = {}
   })
 
-  it('renders', () => {
-    expect(wrapper.is('div')).toBe(true)
-  })
+  describe('shallowMount', () => {
+    const Wrapper = () => {
+      return shallowMount(Badges, { propsData })
+    }
 
-  it('has class "hc-badges"', () => {
-    expect(wrapper.contains('.hc-badges')).toBe(true)
-  })
+    it('renders', () => {
+      expect(Wrapper().is('div')).toBe(true)
+    })
```


unnecessary test case... if it doesn't render all other tests would fail